eta-meeting-organizer-frontend/src/app/meeting-room/components/meeting-room.component.ts

```diff
@@ -1,77 +1,58 @@
 import { Component } from '@angular/core';
-import { FormControl, FormGroup } from '@angular/forms';
-import { TranslateService } from '@ngx-translate/core';
+import { Observable } from 'rxjs';
+import { ApiCommunicationService } from '~/app/shared/services/api-communication.service';
+import { MeetingRoom } from './../../models/meetingroom.model';
 
 @Component({
-  selector: 'app-welcome-component',
+  selector: 'app-meetingRoom',
```

Comment:
```suggestion
  selector: 'app-meeting-room-listing',
```
ha nem egy meeting room hanem egy lista akkor legyen egyértelmü és kebap case

Reply:
javitva 'app-meeting-room-listing',-re